types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
If I could have a question @amritk , how do you have installed 2 versions of `@types/stripe`. As far I know it's not possible to have 2 versions alongside in package.json (see: https://github.com/npm/npm/issues/5499)

The only way to have declared interfaces extended and included in one project is to release something like `@types/stripe` and `@types/stripe-v2`. Then it will be possible to have nice way of extending interface. Now I'm not sure how you have 2 versions of @types/stripe in a project...